app/controllers/openid_application_controller.rb

```diff
@@ -139,13 +138,14 @@ def fetch_aud
   def fetch_smart_launch_context
     response = RestClient.get(Settings.oidc.smart_launch_url,
                               { Authorization: 'Bearer ' + token.token_string })
-    unless response.nil? || response.code != 200
+    raise error_klass('Invalid launch context') if response.nil?
+
+    if response.code == 200
       json_response = JSON.parse(response.body)
       json_response['launch']
     end
-  rescue => e
-    log_message_to_sentry('Error retrieving smart launch context for OIDC token: ' + e.message, :error)
-    nil
+  rescue
+    raise error_klass('Invalid launch context')
```

Comment:
I think we should continue to log the error. It should be unexpected for this to occur (outside of our dev/staging/sandbox environment) and having a report in sentry to track down the root cause would be beneficial.